### PR TITLE
add niransnarayanan.json

### PR DESCRIPTION
**Pull request by nirans2002**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x ] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

https://niransnarayanan-web.web.app/

### CODE REVIEW

**Review by MaskDuck — DISMISSED** [review]

File: domains/niransnarayanan.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "description": "Personal website",
+    "owner": {
+        "username": "nirans2002",
+        "email": "nirans2002@gmail.com"
+    },
+    "record": {
+        "CNAME": "niransnarayanan-web.web.app"
```
[the remark this answers is not in the document]
Comment (nirans2002) [reply domains/niransnarayanan.json:8]: it actually works for me.
image

if there is smthng that i need to, please let me know

**Review by triach-rold — APPROVED** [review]
Reachable and complete.

File: domains/niransnarayanan.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (CuteDog5695) [reply domains/niransnarayanan.json:8]: Site also works for me, I don't see anything wrong with this PR

**Review by iostpa — APPROVED** [review]
Seems good to me

**Pull request merged**